internal/lsp/command.go

```diff
@@ -0,0 +1,19 @@
+package lsp
+
+func toAnySlice(a []string) []any {
+	b := make([]any, len(a))
+	for i := range a {
+		b[i] = a[i]
+	}
+
+	return b
+}
+
+func FmtCommand(args []string) Command {
+	return Command{
+		Title:     "Format using opa-fmt",
+		Command:   "regal.fmt",
```

Comment:
Would you happen to know if there's a well-known command name or some other mechanism at play when an IDE/editor does "format on save" via LSP? What I'm wondering is what we'd need to do to have that just work. If it's command-name-based, we'd need to adjust this. But I know way too little about LSP myself 😅

Reply:
https://zed.dev/docs/configuring-zed#lsp this makes me think there's no convention...

Reply:
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_formatting

And yeah, that'll be trivial to add with this code in place as the response is the same.